exe/ruby-lsp

```diff
@@ -90,4 +94,14 @@ if options[:debug]
   end
 end
 
+if options[:time_index]
+  require "benchmark"
+
+  index = RubyIndexer::Index.new
+
+  result = Benchmark.realtime { index.index_all }
+  puts "Indexing took #{result.round(2)} seconds"
```

Comment:
I feel if we put the version into this message, we can more easily use it in issue reports too?

Reply:
For small projects, I feel the smaller decimals won't really be useful?
LIke these are the 3 consecutive runs on Ruby LSP itself:

```
Ruby LSP 0.16.7: Indexing took 0.83 seconds
Ruby LSP 0.16.7: Indexing took 0.86 seconds
Ruby LSP 0.16.7: Indexing took 0.81 seconds
```

Anything after the 2nd decimal feel more like noise?

Reply:
I increased it to 5 anyway, let's see how it works first.